apps/dashboard/src/app/nebula-app/(app)/components/Chats.tsx

```
@@ -422,3 +456,55 @@ function StyledMarkdownRenderer(props: {
+      });
+      setFeedback(rating);
+      setThankYou(true);
+    } catch (e) {
+      // handle error
+    } finally {
+      setLoading(false);
+    }
+  }
+
+  if (thankYou) {
+    return <div className="mt-2 text-xs text-muted-foreground">Thank you for your feedback!</div>;
```

Biome (1.9.4)

[error] These CSS classes should be sorted.

Unsafe fix: Sort the classes.


(lint/nursery/useSortedClasses)